Command/NotifyDeploymentCommand.php

```diff
@@ -108,9 +107,9 @@ public function performRequest(string $api_key, string $payload): array
 
         $context = [
             'http' => [
-                'method' => 'POST',
-                'header' => \implode("\r\n", $headers),
-                'content' => $payload,
+                'method'        => 'POST',
```

Comment:
IMHO, we should remove this rule: Aligning `=>` array (or `=` assignments) will ad some noise on pull requests when a new entry requires to realign previous entrys
```diff
- 'foo' => 'foo',
- 'bar' => 'bar',
- 'baz' => 'baz',
+ 'foo'    => 'foo',
+ 'bar'    => 'bar',
+ 'baz'    => 'baz',
+ 'foobar' => 'foobar',
```